core/src/rpc_service.rs

```diff
@@ -218,7 +221,18 @@ impl RequestMiddleware for RpcRequestMiddleware {
                 };
             }
         }
-        if self.is_get_path(request.uri().path()) {
+
+        if let Some(result) = process_rest(&self.bank_forks, request.uri().path()) {
+            RequestMiddlewareAction::Respond {
+                should_validate_hosts: true,
+                response: Box::new(jsonrpc_core::futures::future::ok(
+                    hyper::Response::builder()
+                        .status(hyper::StatusCode::OK)
+                        .body(hyper::Body::from(result))
+                        .unwrap(),
+                )),
+            }
+        } else if self.is_get_path(request.uri().path()) {
```

Comment:
Now that we have additional GET methods, it might be helpful to rename `get()` and `is_get_path()` to be more explicit

Reply:
Sure, does 17fe36ac1cd17c5df0a26ee893a3dd70cdff1fe4 look better?

Reply:
Yes, much. Thank you!